### PR TITLE
Display WCA logo when sharing on social media.

### PR DESCRIPTION
**Pull request by AlbertoPdRF**
:warning: I'm not quite sure how to test this locally. I tried with [ngrok](https://ngrok.com/) and [Facebook debugger](https://developers.facebook.com/tools/debug/sharing/) but for some reason it doesn't seem to work.

This is a small step towards fixing #2479. The idea is to at least display something while we wait for someone to fully fix the issue.

### CODE REVIEW

File: WcaOnRails/app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -8,6 +8,8 @@
   <link rel="icon" href='<%= asset_path("android-touch-icon.png") %>'/>
   <link rel="apple-touch-icon" href='<%= asset_path("apple-touch-icon.png") %>'/>
 
+  <meta name="og:image" content="https://www.worldcubeassociation.org/assets/og-wca_logo.png">
```
Comment (jfly) [inline WcaOnRails/app/views/layouts/application.html.erb]: Please use `asset_path` here instead (or `asset_url` if it must be a full url). It's silly for people developing locally to be pointing at our production server.

**Comment by jfly** [comment]
Btw, thanks for looking into this, @AlbertoPdRF!!

File: WcaOnRails/app/views/layouts/application.html.erb (hunk 1 above)
Comment (AlbertoPdRF) [reply WcaOnRails/app/views/layouts/application.html.erb]: Ok! I'll make that change right now.

**Comment by AlbertoPdRF** [comment]
1. Yes, this could definitely be tested if it was deployed to staging, as it is a public server. I don't think Facebook will punish us for this if it doesn't work.
2. No, [Open Graph doesn't support SVG images](https://indieweb.org/The-Open-Graph-protocol#Does_not_support_SVG_images), so I just created a white background of the [recommended size](https://developers.facebook.com/docs/sharing/best-practices#images) and superimposed the logo.

**Comment by jfly** [comment]
> No, Open Graph doesn't support SVG images, so I just created a white background of the recommended size and superimposed the logo.

Damn. Could you update your commit message with instructions about how you created this image, so if we ever want to change it, people know how to recreate it?

**Comment by moralsh** [comment]
As this only affects the generated HTML, one easy way to test it is just save an html page from your development server and upload it somewhere public (like github pages) and pass that URL to the open graph debugger.

another concern is that maybe the PNG should have transparent background, an easy way to achieve this is using https://cloudconvert.com/svg-to-png.

Hope this helps :)

**Comment by AlbertoPdRF** [comment]
After testing on staging, this seems to work! I amended the commit message to keep a record of when and why the image `og-wca_logo.png` appeared.

**Pull request merged**